enum/Analytics.ts

```diff
@@ -8,6 +8,7 @@ export enum EventName {
 	AGENT_TRANSFER_SUCCESSFUL = 'agent_transfer_successful',
 	CHAT_CLOSED_BY_AGENT = 'chat_closed_by_agent',
 	CHAT_CLOSED_BY_TIMEOUT = 'chat_closed_by_timeout',
+	AGENT_TRANSFER_FAILED_DUE_TO_DISCONNECTED_AGENT = 'chat_closed_due_to_disconnected_agent',
```

Comment:
This is an agent disconnected event, not an agent transfer event?

Reply:
We currently have some chatclosed handlers after we catch a message with AgentDisconnect type. Do we want to remove them?

Reply:
@ear-dev Clarifcation. I added them in this PR. Removed them later, as per @Shailesh351 instructions.